lib/model/course/courses_mock.dart

```diff
@@ -10,17 +10,21 @@ class CoursesMock implements Courses {
 
   CoursesMock() {
     for (int i = 1; i < GENERATE_COURSES; i++) {
-      courses.add(new Course(
-          generateMockCourseTitle(i),
-          generateMockDepartment(i),
-          generateMockTime(i),
-          generateMockDescription(i),
-          generateMockClassLength(i),
-          generateMockECTS(i),
-          generateMockEmail(i),
-          generateMockName(i),
-          generateMockAvailability(i),
-          generateMockFavorite(i)));
+      courses.add(
```

Comment:
I forgot if I mentioned this but with the builder pattern these parameters dont need to be specified in any specific order so it'll be nice for the future if we add/delete/modify these.

Reply:
ok got it. Sounds cool